src/utils/task/checkLinkedTaps.js

```diff
@@ -122,9 +122,6 @@ const setupTapTask = async ({ globalConfig, allTasks, command, options }) => {
       params: { tap: command }
```

Comment:
So the reason you can remove `line 26` below is because of this line here.
In the past, before custom tap tasks where a thing
  * The `options` array was always parsed into the `params` object from the same location => `runTask.js - line 30`
    * `runTask.js` calls `executeTask` which is where all `options` were parsed into the `params` object 
    * This call comes after the task to run is already found.
  * The `tap name` was injected into the `options` here, the `params` object would have it when the `options` are parsed

With injected taps and custom tap tasks being added
  * I had to move the params parsing into this file
    * Which sucks cause it's not all happening in one place, but does allow injected taps / custom tasks to work
  * We needed to know the `params.env value` when calling the `injectedService` on `line 174` of this file
    * This ensure that when  `yaml` and `.env` files are loaded, it uses the correct environment to find the files to load
    * For example it allows loading `~/.kegConfig/keg-herkin.development.env` when the environment is `development`
    * Without parsing the options, there's no way to know `--env development` exists

All of that being said, removing this line should be ok, assuming none of the task expect the `tap=<tap-name>` option to exist.
* I believe this is the case, as I tried not to ever access the options after they have been parsed while writing the tasks.
* I did a double check I don't see it being used anywhere so it should be fine. But I might have missed it, so you might want to do double check as well.

Reply:
Actually, I did find one location the options are being used, in the `destroyService`.
  * It's checking for the `image` argument using the `optionsHasArg` helper, so it won't break anything
  * But honestly I'm not a fan of this. There should be better way.
  * Can you do me a favor and add a `TODO` to the `line 14` of the `destroyService` to remove that
    * We should also remove the `optionsHasArg` helper
    * We really should not be depending on the `options` array

Reply:
Okay yeah that makes more sense now